src/QsCompiler/CommandLineTool/Commands/Build.cs

```diff
@@ -89,9 +112,8 @@ internal static bool IncorporateResponseFiles(BuildOptions options, out BuildOpt
             }
         }
 
-
```

Comment:
I actually like trimming the double-lines. I've never been a big fan of them as, to me, they add a jaggedness to the formatting of a file.

Reply:
This is [SA1507](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1507.md). I don't have a strong preference, but I slightly prefer not having double lines (like @ScottCarda-MS). IMO, if a file is large enough that it would benefit from sections separated by double lines, it might be better to create separate files...

Reply:
I also have a slight preference for trimming double lines. As @samarsha mentioned, if the large is long enough for sections to be separated by double lines, it might be a good idea to create a separate file.

Reply:
works for me.